pkg/asset/installconfig/gcp/gcp.go

```diff
@@ -103,13 +130,25 @@ func selectRegion(project string) (string, error) {
 	sort.Strings(shortRegions)
 
 	defaultRegion := "us-central1"
+	foundIdx := 0
+	for idx := range shortRegions {
+		if shortRegions[idx] == defaultRegion {
+			foundIdx = idx
+			break
+		}
+	}
+
+	if foundIdx >= 0 {
+		defaultRegion = validRegions[0]
+	}
```

Comment:
1. Is this defensive programming or is it possible the list of regions would not have `us-central1`? 
2. A more idiomatic way of writing this would be something like
```suggestion
	var found int
	for i, v := range shortRegions {
		if v == defaultRegion {
			found = i
			break
		}
	}
```